packages/cyberstorm/src/components/CopyButton/CopyButton.tsx

```diff
@@ -0,0 +1,72 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faClone } from "@fortawesome/free-regular-svg-icons";
+import { faCheck } from "@fortawesome/free-solid-svg-icons";
+import React, { PropsWithChildren, useEffect } from "react";
+import { useState } from "react";
+import styles from "./CopyButton.module.css";
+import { Tooltip } from "../Tooltip/Tooltip";
+
+type CopyButtonProps = {
+  text: string;
+};
+type CopiedValue = string | null;
+type CopyFn = (text: string) => Promise<void>;
+
+function useCopyToClipboard(): CopyFn {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [copiedText, setCopiedText] = useState<CopiedValue>(null);
```

Comment:
If `copiedText` is never used (as per the eslint comment above), why set it's value at all? Seems this isn't actually used in any way?

If you yeet this `useState`, this hook just returns a function. In which case it doesn't need to be a hook? Why not just define `copy` as a regular function?